tests/visco_plastic_yield_strain_weakening.prm

```diff
@@ -0,0 +1,239 @@
+
+# Initial temperature field
+# Typical continental geotherm based equations 4-6 from Chapman 1986 (Geol. Soc. Lon.)
+# The initial constraints are:
+#   surface temperature (ts) = 273 K; basal temperature (tb) = 1573 K; Moho
+#   temperature (tm) = 873 K; crust heat production (A) = 9.e-7 W/m^3; crust
+#   thermal conductivity (kc) = 2.5 (W/(m K)); 
+# To satisfy these constraints, the following values are required:
+#   surface heat flow (qs) = 0.0635 W/m^2; basal heat flow = 0.365 W/m^2;
+#   mantle thermal conducitivty (km) = 3.65 (W/(m K)); 
+subsection Initial conditions
+  set Model name = function
+  subsection Function
+    set Variable names = x,y
+    set Function constants = h=100e3,ts=273,tm=873.,qs=0.0635,qb=0.0365,kc=2.5,km=3.65,A=9.e-7
+    set Function expression = if( (h-y)<=30.e3, \
+                                  ts + (qs/kc)*(h-y) - (A*(h-y)*(h-y))/(2.0*kc), \
+                                  tm + (qb/km)*(h-y-30.e3))                             
+  end
+end
+
+# Internal heating (only internal heating for crust)
+# Assuming a very large decay time, the heat production (W/m^3) is effectively:
+#   radioactive_heating_rate (W/kg) * density (kg/m^3)
+# For a reference crustal density of 2800 kg/m^3 and 9.e-7 W/m^3 heating production,
+# the heating rate (assuming a single element) is 9.e-7/2800. = 3.214287e-10.
+subsection Heating model
+  set Model name = radioactive decay
+  subsection Radioactive decay
+    set Number of elements    = 1
+    set Heating rates                 = 3.2142857e-10
+    set Half decay times              = 1.e20
+    set Initial concentrations mantle = 0.0
+    set Initial concentrations crust  = 1.0 
+    set Crust defined by composition  = true
+    set Crust composition number      = 0
+  end
+end
+
+
+# Material model (values for background material & compositional fields)
+# Note that since we now are tracking finite strain compositional fields,
+# (3 components in 2D), properties must be specified for each of these
+# fields.  However, the material model does not use these values in 
+# determining the element properties given the finite strain compositional
+# fields are listed first.  
+subsection Material model
+  set Model name = visco plastic
+
+  subsection Visco Plastic
+
+    set Reference temperature = 293
+
+    # As the boundaries are all free-slip, so use the reference strain-rate
+    # (used on first nonlinear iteration) to specify the strain-rate used
+    # in the viscous flow law.  Set the minimum strain-rate to the same value.
+    set Minimum strain rate = 1.e-20
+    set Reference strain rate = 1.e-20
+
+    # Minimum, maximum and reference viscosity 
+    set Minimum viscosity = 1e18
+    set Maximum viscosity = 1e28
+    set Reference viscosity = 1e22
+
+    # Thermal diffusivity is adjusted to match thermal conductivities
+    # assumed in the assigning the initial geotherm
+    set Thermal diffusivities = 1.474747e-6,1.190476e-6,1.190476e-6,1.190476e-6,1.190476e-6
```

Comment:
I see that it does not matter, because they are zeroed in the material model anyway, but personally I would think it is more intuitive to give the 'strain' fields the same properties as the background material. But this is up to you, if this is only a test it does not matter much.